### DISCUSSION TITLE
Admin user can create Super admin user

### DISCUSSION THREAD

**Discussion by fatoon29** [directus/directus#7619] [Archive]
In my app, there is a super admin role, admine role , and two other role.
When a admin create an user, i do'nt want that in the role selection, he can choose the super admin role.
How can i take off from the list a role?

I manage to make a validation field but i don't really like this solution because of the error message ("this value can't be 82776421-0cf6-4286-a25f-5feb31af095d")  : 

```
{
    "_and": [
        {
            "role": {
                "_neq": "82776421-0cf6-4286-a25f-5feb31af095d"
            }
        },
        {
            "role": {
                "_neq": "e510874b-1969-4173-87c4-896de035bae2"
            }
        },
        {
            "role": {
                "_neq": "4531d801-bc3c-4228-b4ac-b2b86759ca2c"
            }
        }
    ]
}
```

**Comment by rijkvanzanten** [comment]
You could add that same permission to the `read` permissions to prevent that role option from showing up in the first place 🤔

**Reply by fatoon29** [reply]
Was an easy solution when i read your response 😓
That work really fine, thanks
